packages/babel-parser/ast/spec.md

```diff
@@ -108,7 +108,6 @@ These are the core @babel/parser (babylon) AST node types.
   - [ClassExpression](#classexpression)
```

Comment:
Is this file auto-generated?

Reply:
No. It seems to me this file is modified from the estree spec. We could somehow generate it if we move the labelling/descriptions to source comments.